translations/messages-en.properties

```diff
@@ -995,4 +995,10 @@ configuration.role.offline = Offline
 configuration.role.offline.warning = Users with offline access can view and update documents without an internet connection and can only view a portion of the data. Online users can access all documents but must have an internet connection at all times.
 configuration.roles = Roles
 configuration.roles.add = Add new role
-configuration.authorization = Roles & permissions
\ No newline at end of file
+configuration.authorization = Roles & permissions
+contact.last.visited.date = Visited {{date}}
+contact.last.visited.unknown = Last visited unknown
+contacts.results.sort = Sort results
+contacts.results.sort.title = Sort this list
+contacts.results.sort.alpha = Alphabetically
+contacts.results.sort.dlv = By date last visited
```

Comment:
I think it's better not to use the acronym "dlv" here - maybe just `contacts.results.sort.last.visited` instead?